client/lcd/lcd_test.go

```diff
@@ -665,14 +671,17 @@ func doUnbond(t *testing.T, port, seed, name, password string, delegatorAddr, va
 		"account_number": %d,
 		"sequence": %d,
 		"gas": 10000,
-		"delegate": [],
-		"unbond": [
+		"delegations": [],
+		"begin_unbondings": [
 			{
 				"delegator_addr": "%s",
 				"validator_addr": "%s",
 				"shares": "30"
 			}
-		]
+		], 
+		"complete_unbondings": [], 
+		"begin_redelegates": [], 
+		"complete_redelegates": []
```

Comment:
`chain_id` might be needed here (can't see entire JSON struct in diff)

Reply:
same as above